frontstage_api/resources/passwords/request_password_change.py

```diff
@@ -5,23 +5,24 @@
 from flask_restplus import Resource, fields
 from structlog import wrap_logger
 
-from frontstage_api import api, auth
+from frontstage_api import auth
 from frontstage_api.controllers import django_controller, party_controller
+from frontstage_api.resources import passwords_api
 
 
 logger = wrap_logger(logging.getLogger(__name__))
 
-user_details = api.model('UserDetails', {
+user_details = passwords_api.model('UserDetails', {
     'username': fields.String(required=True),
 })
 
 
-@api.route('/request-password-change')
+@passwords_api.route('/request-password-change')
 class RequestPasswordChange(Resource):
 
     @staticmethod
     @auth.login_required
```

Comment:
We can now add the authentication header to the namespaces instead of the individual endpoints. Not sure if you want to do it part of this pull request.

Reply:
I'm looking into that now, it kind of looks like there's no @before_first_request for api/namespace objects as far as I can see so not sure if we can do that

Reply:
^ Lies ^ I just found a way to do it, will introduce now

Reply:
^ It didn't work, you're supposedly able to specify decorators to run on all resources when you define the namespace, i.e. 
``Namespace(..., decorators=[decorator])`` 
But that didn't work for me. You can specify them like that in the Api definition though so not sure if it's a bug or something I'm missing.